server/db/driver/pg/pg.go

```diff
@@ -116,6 +141,40 @@ func NewArchiver(ctx context.Context, cfg *Config) (*Archiver, error) {
 		return nil, err
 	}
 
+	switch cfg.Net {
+	case asset.Mainnet:
+		archiver.keyParams = chaincfg.MainNetParams()
+	case asset.Testnet:
+		archiver.keyParams = chaincfg.TestNet3Params()
+	case asset.Simnet:
+		archiver.keyParams = chaincfg.SimNetParams()
+	default:
+		return nil, fmt.Errorf("unkown network %d", cfg.Net)
+	}
+
+	// Get the master extended public key.
+	masterKey, err := hdkeychain.NewKeyFromString(cfg.FeeKey, archiver.keyParams)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing  master pubkey: %v", err)
+	}
+
+	// Get the external branch (= child 0) of the extended pubkey.
+	archiver.feeKeyBranch, err = masterKey.Child(0)
+	if err != nil {
+		return nil, fmt.Errorf("error creating external branch: %v", err)
+	}
+
+	// Get a hash of the serialized secp256k1.PublicKey to create an identifying
+	// hash of this key for the child counter DB table.
+	pubKey, err := archiver.feeKeyBranch.ECPubKey()
+	if err != nil {
+		return nil, fmt.Errorf("error getting pubkey from extended key: %v", err)
+	}
+	archiver.keyHash = dcrutil.Hash160(pubKey.Serialize())
```

Comment:
No reason we have to use the hash of the derived pubkey for the database id. I just want to avoid storing the raw extended key. I'm fairly certain that a simple `dcrutil.Hash160([]byte(cfg.FeeKey))` would be just as unique and is simpler.

Reply:
Sounds alright to me.  Just wondering why hash160 though, space savings?  Seems like a single blake256 would do it too.  I don't have much expertise here, but my understanding is that ripemd160 is being avoided in decred code where possible.  @jrick opinion about suitability of hash160 for the purposes of a DB key?

Reply:
Yeah. Just a little smaller.